civic-tech-club/src/App.js

```diff
@@ -0,0 +1,21 @@
+
+import './App.css';
```

Comment:
I've never used typescript. I enabled it for this project because I'd like to learn it, but there may be some things in here that need to change to use it properly

Reply:
Happy to chat about it if you're interested!